fuzz/fuzz_targets/common.rs

```diff
@@ -0,0 +1,13 @@
+use std::io::{Error, Write};
+
+pub struct Null;
+
+impl Write for Null {
```

Comment:
I was looking for something like this before I implemented it, I saw `sink` in the documentation, but glanced over it because I thought that something like this would surely be named in a different way :-) Thank you.